js/src/launchAsync.ts

```diff
@@ -116,6 +116,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             }
         };
 
+        options.parent = options.parent === null ? document.body : options.parent;
```

Comment:
checking for null is not the same as check for validity/existence. Am thinking along the lines of:

document.getElementById(parent)

to check if it really exists

Reply:
done, 
parent is validated against undefined, if not undefined then check if existis in document, if not exists then set document.body